.github/workflows/tests.yml

```diff
@@ -45,14 +45,14 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ['3.7', '3.8']
+        python-version: ['3.9', '3.10']
```

Comment:
Could we get the most out of our testing time by doing 3.7 and 3.10 (low and high version #s)?

Reply:
I like this idea